go.mod

```diff
@@ -11,7 +11,8 @@ require (
 	github.com/go-kit/kit v0.9.0
 	github.com/google/uuid v1.1.1
 	github.com/lestrrat-go/jwx v0.9.0
-	github.com/optimizely/go-sdk v1.2.0
+	// Using custom go-sdk branch for testing purposes.
+	github.com/optimizely/go-sdk v1.1.3-0.20200604162003-16569aea2cdc
```

Comment:
I think we should first create a Go SDK release that includes the new method, then update the dependency here to the new Go SDK version.